ballerina/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "grpc"
-version = "1.2.3"
+version = "1.3.0"
 distribution = "2201.0.1"
```

Comment:
```suggestion
distribution = "2201.1.0"
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/797/commits/fc6df8c0cc28db1bfd3e5c8bbbba82fd41ee5f9c